app/components/windows/AdvancedStatistics.tsx

```diff
@@ -0,0 +1,194 @@
+
+  private updateInterval = 0;
+  private notifications: INotification[] = [];
+  private streamingStatus: EStreamingState = null;
+  private notificationPushed: Subscription = null;
+  private streamingStatusChange: Subscription = null;
+
+  mounted() {
+    this.notificationPushed = this.notificationsService.notificationPushed.subscribe(notify => {
+      this.onNotificationHandler(notify);
+    });
+
+    this.streamingStatusChange = this.streamingService.streamingStatusChange.subscribe(status => {
+      this.onStreamingStatusChange(status);
+    });
+
+    this.notifications = this.notificationsService
+      .getAll()
+      .filter(notification => notification.subType !== ENotificationSubType.DEFAULT);
+    // update the time labels
+    this.updateInterval = window.setInterval(() => {
+      this.$forceUpdate();
+    }, 60 * 1000);
+  }
+
+  destroyed() {
+    this.notificationsService.markAllAsRead();
+    this.notificationPushed.unsubscribe();
+    this.streamingStatusChange.unsubscribe();
+    clearInterval(this.updateInterval);
+  }
+
+  get streamQuality() {
+    return this.performanceService.streamQuality;
+  }
+
+  get status(): { type: string; description: string } {
+    if (this.streamingStatus === 'reconnecting' || this.streamQuality === EStreamQuality.POOR) {
+      return {
+        type: 'error',
+        description: $t('Streamlabs OBS is experiencing difficulties broadcasting'),
+      };
+    }
+
+    if (this.streamQuality === EStreamQuality.FAIR) {
+      return {
+        type: 'warning',
+        description: $t('Streamlabs OBS is experiencing minor issues.'),
+      };
+    }
+
+    return {
+      type: 'success',
+      description: $t('Streamlabs OBS is running normally'),
+    };
+  }
+
+  onStreamingStatusChange(status: EStreamingState) {
+    this.streamingStatus = status;
+  }
+
+  moment(time: number): string {
+    return moment(time).fromNow();
+  }
+
+  onNotificationHandler(notification: INotification) {
+    if (notification.subType === ENotificationSubType.DEFAULT) {
+      return;
+    }
+    this.notifications.push(notification);
+  }
+
+  get statusIcon() {
+    if (this.status.type === 'error') {
+      return <span class="fa fa-minus-circle" />;
+    }
+
+    if (this.status.type === 'warning') {
+      return <span class="fa fa-exclamation-triangle" />;
+    }
+
+    if (this.status.type === 'success') {
+      return <span class="fa fa-check" />;
+    }
+  }
+
+  get statusBar() {
+    return (
+      <div>
+        <h2>Status</h2>
+        <div
+          class={cx(
+            styles.status,
+            this.status.type === 'error' ? styles.error : '',
+            this.status.type === 'warning' ? styles.warning : '',
+            this.status.type === 'success' ? styles.success : '',
+          )}
+        >
+          {this.statusIcon}
+          <span>{this.status.description}</span>
+        </div>
+      </div>
+    );
+  }
+
+  notificationsArea() {
```

Comment:
should be a getter if there are no parameters accepted

Reply:
If I make it a getter, the setInterval forceUpdate that makes the `2 minutes ago` labels update doesn't work

Reply:
Making it a getter breaks the moment driven time label updates